server/lib/configuration.js

```diff
@@ -149,6 +149,33 @@ var conf = module.exports = convict({
     doc: 'Google Analytics id',
     format: String
   },
+  hpkp: {
+    enabled: {
+      default: true,
+      doc: 'Feature flag for appending HPKP headers',
+      env: 'HPKP_ENABLED'
+    },
+    includeSubDomains: {
+      default: true,
+      doc: 'Include Sub-Domains',
+      env: 'HPKP_INCLUDE_SUBDOMAINS'
+    },
+    max_age: {
+      default: 1,
+      doc: 'Max age for HPKP headers (seconds)',
+      env: 'HPKP_MAX_AGE'
+    },
+    pin_sha256: {
+      default: [
+        '5kJvNEMw0KjrCAu7eXY5HZdvyCS13BbA0VJG1RSP91w=',
```

Comment:
OK, I've gone back and forth on this in my mind and have settled since last week. Since incorrect values cause such problems, I do not think the default values should be here, so we avoid accidents when deploying with ssl to environments we are not yet familiar with. For prod, these should be set by @jbuck and @jrgm in puppet-config, and in fxa-dev, we can set them as values in the .yml values.